luapgsql.c

```diff
@@ -32,7 +32,16 @@
 #define htobe64(x) OSSwapHostToBigInt64(x)
 #elif __linux__
 #include <endian.h>
+#elif _WIN32
+#include <sys/param.h>
+#if BYTE_ORDER == LITTLE_ENDIAN
+#include <x86intrin.h>
```

Comment:
I think `__builtin_bswap64` is a compiler builtin. You shouldn't need this include?

Reply:
You are right. I removed it.